ui/apps/platform/eslint-plugins/pluginGeneric.js

```diff
@@ -0,0 +1,139 @@
+/* globals module */
+
+const rules = {
+    // ESLint naming convention for positive rules:
+    // If your rule is enforcing the inclusion of something, use a short name without a special prefix.
+
+    'ExternalLink-a': {
+        // Require ExternalLink and a elements for consistent presentation.
+        // For example, ExternalLinkAltIcon follows a element.
+        meta: {
+            type: 'problem',
+            docs: {
+                description: 'Require ExternalLink and a elements for consistent presentation',
```

Comment:
Nit what do you think about either
```suggestion
                description: 'Require ExternalLink and <a> elements for consistent presentation',
```
or
```suggestion
                description: 'Require ExternalLink and anchor elements for consistent presentation',
```

Reply:
Thank you! Yes, **anchor**  for less punctuation.